01-introduction.md

```diff
@@ -74,11 +75,19 @@ so that they get the positive feedback of being able to read whole stories as ea
 The basic idea is that once children find reading rewarding,
 they'll be motivated to go back and figure out the letters.
 
-Both approaches are consistent with what we know about how children learn,
-so we need to study them in action to determine which is better.
-The findings have been contradictory:
-while some studies find that phonics works best,
-others show that whole language yields better results.
+These approaches are not equally consistent with what we know about how children
+learn to read. While there is some evidence that proficient, adult readers
+identify whole word patterns, the scientific consensus among linguists,
+psychologists and neuroscientists is that in order for a child's brain to learn
+how to read, it is required to first make out the mapping of sounds to letters.
```

Comment:
On Sat, Jan 16, 2016 at 12:50:59PM -0800, Ariel Rokem wrote:
> > So some kids are learning from logograms (they just aren't as good
> > at consonant addition and deletion tasks).
>
> I actually see that as evidence in favor of a strict phonics
> view. If you learn an alphabetical system (=one that translates
> phonemes to symbols), you will develop the phonological awareness
> required for skilled reading.

Folks have been reading purely logographic systems for a long time.
I'd be surprised if those systems produced zero skilled readers.

> If you are learning a logographic system, you will not implicitly
> learn about symbol <=> sound associations, necessary for skilled
> reading [citation needed]. But I am still not sure.

And yeah, in the absence of cited studies, I'd err on the side of
wiggle words ;).